### PR TITLE
Feature/ui styling tracker complex schema

### PR DESCRIPTION
**Pull request by astewes**
-  Adds nested tab layout for complex schema

hydrator

tracker

### CODE REVIEW

File: cdap-ui/app/directives/complex-schema/complex-schema-helpers.js [hunk 1]
```diff
@@ -62,12 +62,26 @@ angular.module(PKG.name+'.commons')
...
       displayType: type,
       type: storedType,
-      nullable: nullable
+      nullable: nullable,
+      nested: checkComplexType(type)
     };
   }
 
+  function checkComplexType(displayType) {
+    if (displayType === 'array' ||
```
Comment (ajainarayanan) [inline cdap-ui/app/directives/complex-schema/complex-schema-helpers.js]: Can this be 

```
var complextTypes = ['array', 'enum', 'map', 'record', 'union'];
(complexTypes.indexOf(displayType) !== -1) ? true : false;
```

File: cdap-ui/app/directives/complex-schema/embedded-schema-selector/embedded-schema-selector.html [hunk 2]
```diff
@@ -15,7 +15,8 @@
 -->
 
 <div>
-  <div ng-if="['enum', 'array', 'map', 'union', 'record'].indexOf(Embedded.displayType) !== -1">
+  <div ng-if="['enum', 'array', 'map', 'union', 'record'].indexOf(Embedded.displayType) !== -1"
```
Comment (ajainarayanan) [inline cdap-ui/app/directives/complex-schema/embedded-schema-selector/embedded-schema-selector.html]: Is it possible for us to reuse `SchemaHelpers. checkComplexType` here?

**Comment by astewes** [comment]
Bamboo build: http://builds.cask.co/browse/CDAP-DRC4088-6

**Comment by ajainarayanan** [comment]
👍 LGTM

**Pull request merged**